spec/support/bootstrap/db_config.rb

```diff
@@ -63,10 +63,10 @@ def default_connection_prefix(db_type)
   end
 
   def default_name
-    if ENV['TEST_ENV_NUMBER']
-      "cc_test_#{ENV['TEST_ENV_NUMBER']}"
-    else
+    if ENV['TEST_ENV_NUMBER'].nil? || ENV['TEST_ENV_NUMBER'].empty?
```

Comment:
I think the most concise way to write the check is `ENV['TEST_ENV_NUMBER'].presence` (from Rails' Active Support). If the key does not exist, this returns `nil`, if the key exists, but contains an empty string, also `nil` is returned, otherwise the string is returned.

Reply:
Updated to use this instead